backend/src/main/java/io/papermc/hangar/db/dao/v1/VersionsApiDAO.java

```diff
@@ -104,10 +104,10 @@ WHEN exists(SELECT * FROM pinned_versions piv WHERE piv.version_id = pv.id AND l
                    <endif>)
                    AND
                <endif>
-               lower(p.slug) = lower(:slug) AND
-               pv.version_string = :versionString
+               p.id = :projectId AND
+               pv.id = :versionId
     """)
-    @Nullable Map.Entry<Long, Version> getVersionWithVersionString(String slug, String versionString, @Define boolean canSeeHidden, @Define Long userId);
```

Comment:
the join on projects isn't necessary anymore

Reply:
I think this can also just be replaced with getVersion which can be simplified itself